roles/download/defaults/main.yml

```diff
@@ -710,5 +716,5 @@ download_defaults:
   version: None
   url: None
   unarchive: false
-  owner: kube
+  owner: None
```

Comment:
None ?

Reply:
Left over from a test. I changed it back to kube just now.